### PR TITLE
Setup basic Rpc framework

### PR DESCRIPTION
**Pull request by realwanpengli**
1. Basic protocol definition
2. Rpc Master/Slave
3. Logger

### CODE REVIEW

File: SignalRServiceBenchmarkPlugin/common/Util.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+namespace Common
+{
+    public class Util
+    {
+        public static void CreateLogger(string directory, string name)
+        {
+            Log.Logger = new LoggerConfiguration()
+                .MinimumLevel.Debug()
+                .WriteTo.Console()
```
Comment (ccic) [inline SignalRServiceBenchmarkPlugin/common/Util.cs]: Here write log twice: to console and to file?

File: SignalRServiceBenchmarkPlugin/framework/master/ArgsOption.cs [hunk 2]
```diff
@@ -0,0 +1,22 @@
+﻿using CommandLine;
+using System.Collections.Generic;
+
+namespace Rpc.Master
+{
+    public class ArgsOption
+    {
+        // Rpc 
+        [Option("RpcPort", Required = false, Default = 5050, HelpText = "")]
```
Comment (ccic) [inline SignalRServiceBenchmarkPlugin/framework/master/ArgsOption.cs]: Please add HelpText for all fields, especially those have default values. You'd better tell user the default value in help description.

Comment (realwanpengli) [reply SignalRServiceBenchmarkPlugin/framework/master/ArgsOption.cs]: Added

File: SignalRServiceBenchmarkPlugin/framework/master/Program.cs [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+            WaitRpcConnectSuccess(clients);
+        }
+
+        private static ArgsOption ParseArgs(string[] args)
+        {
+            Log.Information($"Parse arguments...");
+            var argsOption = new ArgsOption();
+            var result = Parser.Default.ParseArguments<ArgsOption>(args)
+                .WithParsed(options => argsOption = options)
+                .WithNotParsed(error => { });
```
Comment (ccic) [inline SignalRServiceBenchmarkPlugin/framework/master/Program.cs]: Please handle error, at least print error to console.

**Pull request merged**